unittests/Feature/FeatureModelTransaction.cpp

```diff
@@ -832,4 +832,81 @@ TEST_F(FeatureModelRemoveFeatureTransactionTest,
   ASSERT_EQ(*C->getChildren<Relationship>().begin(), CB->getParent());
 }
 
+//===----------------------------------------------------------------------===//
+//                     FeatureModelTransaction Tests
+//===----------------------------------------------------------------------===//
+
+class FeatureModelTransactionTest : public ::testing::Test {
+protected:
+  void SetUp() override {
+    FeatureModelBuilder B;
+    B.makeFeature<BinaryFeature>("a", true);
+    FM = B.buildFeatureModel();
+    ASSERT_TRUE(FM);
+  }
+
+  std::unique_ptr<FeatureModel> FM;
+};
+
+TEST_F(FeatureModelTransactionTest, addFeatureToModel) {
+  size_t FMSizeBefore = FM->size();
+
+  vara::feature::addFeature(*FM,
+                            std::move(std::make_unique<BinaryFeature>("ab")),
```

Comment:
```suggestion
                            std::make_unique<BinaryFeature>("ab"),
```
`move` is not required here because the created unique_ptr is already an rvalue